config/index.js

```diff
@@ -12,110 +18,132 @@ import {
   FIREBASE_DATABASE_URL,
   FIREBASE_PROJECT_ID,
   FIREBASE_STORAGE_BUCKET,
-  FIREBASE_MESSAGING_SENDER_ID
+  FIREBASE_MESSAGING_SENDER_ID,
+  FAVICON
 } from './names'
-import mainConfig from './main'
+import envConfig from './env.config'
+import networkConfig from './network.config'
 import manifest from '../extension/manifest.json'
 
-const configuration = () => {
-  return mainConfig
+const envConfiguration = () => {
+  return envConfig
 }
 
-const getConfig = ensureOnce(configuration)
+const networkConfiguration = () => {
+  return networkConfig
+}
+
+const getEnvConfig = ensureOnce(envConfiguration)
+
+const getNetworkConfig = ensureOnce(networkConfiguration)
+
+export const getEnviroment = () => {
+  const env = process.env.NODE_ENV
+  return (env === PRODUCTION || env === PRE_PRODUCTION || env === STAGING || env === DEVELOPMENT)
+    ? env
+    : STAGING
+}
+
+export const getNetwork = () => {
+  const env = process.env.NETWORK
```

Comment:
kind of confusing that this is called `env`. Wouldn't `network` be a better name?